inc/enqueue.php

```diff
@@ -51,22 +51,26 @@ function bootscore_scripts() {
 add_action('wp_enqueue_scripts', 'bootscore_scripts');
 
 
-
-/*
- * Register compiled CSS to editor
- */ 
+/**
+ * Register compiled CSS for block editor and Pattern Library
+ */
 function bootscore_add_editor_styles() {
+  // Add support for editor styles and main.css for the editor
   add_theme_support('editor-styles');
   add_editor_style('assets/css/main.css');
 
-  // Check if the current page is the Gutenberg editor and enqueue CSS to the main admin area to use variables in theme.json
-  $screen = get_current_screen();
-  if ($screen && $screen->is_block_editor) {
-    wp_enqueue_style('editor-style', get_stylesheet_directory_uri() . '/assets/css/editor.css', array(), '1.0', 'all');
-  }
-}
-add_action('enqueue_block_editor_assets', 'bootscore_add_editor_styles');
+  // Enqueue additional styles for block editor and Pattern Library
+  add_action('admin_enqueue_scripts', function($hook_suffix) {
+    $screen = get_current_screen();
+    
+    // Enqueue editor.css only in the block editor
+    if ($screen && $screen->is_block_editor) wp_enqueue_style('editor-style', get_stylesheet_directory_uri() . '/assets/css/editor.css');
 
+    // Enqueue main.css only in the Pattern Library
+    if ('appearance_page_edit-wp-patterns' === $hook_suffix) wp_enqueue_style('bootscore-pattern-library-styles', get_stylesheet_directory_uri() . '/assets/css/main.css');
+  });
+}
```

Comment:
Shouldn't this be outside the `after_setup_theme` action?

Reply:
You're right, have splitted the editor and the pattern enqueues in two functions. See https://github.com/bootscore/bootscore/pull/890/commits/769ae8897aca42620a40bd350a3c631b19c38eeb